contracts/BatchExchange.sol

```diff
@@ -291,9 +292,9 @@ contract BatchExchange is EpochTokenLocker {
         require(utility.add(burntFees) > disregardedUtility, "Solution must be better than trivial");
```

Comment:
Is this still needed?  checkProposedObjectiveValue makes sure we are strictly better than the previous solution, right?